docs/docs/guides/labs/components/building-pipelines-with-components/adding-components.md

```diff
@@ -68,9 +68,28 @@ Each `component.yaml` file supports a rich templating syntax, powered by `jinja2
 A common use case for templating is to avoid exposing environment variables (particularly secrets) in your YAML files. The Jinja scope for a `component.yaml` file contains an `env` function that can be used to insert environment variables into the template:
 
 ```yaml
-component_type: my_snowflake_component
+type: snowflake_lib.SnowflakeComponent
 
 attributes:
   account: "{{ env('SNOWFLAKE_ACCOUNT') }}"
   password: "{{ env('SNOWFLAKE_PASSWORD') }}"
+```
+
+#### Multiple documents within a file
```

Comment:
I would call this something like "Multiple component instances" or "Configuring multiple component instances in the same file"